flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/validation/FlinkDeploymentValidator.java

```diff
@@ -24,11 +24,20 @@
 /** Validator for {@link FlinkDeployment} resources. */
 public interface FlinkDeploymentValidator {
```

Comment:
Maybe we could also add a method called `configure(Configuration operatorConf)` that could be very useful in practice